server/mvcc/kvstore_txn.go

```diff
@@ -24,6 +24,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const BigValueSize int = 1 * 1024 * 1024
```

Comment:
It should be a flag  (assuming we need simillar threshold).

Reply:
Have changed to flag.